lib/rest/accounts/v1/credential/aws.js

```diff
@@ -66,34 +66,35 @@ AwsList = function AwsList(version) {
    * @memberof Twilio.Accounts.V1.CredentialContext.AwsList
    * @instance
    *
-   * @param {object|function} opts - ...
+   * @param {object} [opts] - ...
```

Comment:
Why are you removing `function` as a valid type for `opts` here. The argument can be a callback function as well if you are not planning to pass in options

Reply:
`opts` is marked as optional, so conceptually, you can either provide an `object` for `opts` and a function for `callback`, or omit `opts` and provide a `function` for `callback`. Either way, the `callback` is a `function` and `opts` is an `object`.

(Of course, internally, we need to swap them around, but that does not seem like a relevant detail for the consumer.)